src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -44,8 +44,8 @@ class HuiEntityButtonCard extends LitElement implements LovelaceCard {
 
   public static getStubConfig(): object {
     return {
-      tap_action: { action: "more-info" },
-      hold_action: { action: "none" },
+      tap_action: { action: "toggle" },
```

Comment:
We should make sure the entity can be toggled.

Reply:
Hmm, I won't know that at the time I create a stub config. Perhaps it could be a feature of the UI editor though? Not present toggle as an option if entity is not toggleable and make inactive until an entity is selected?